synapse/federation/sender/transaction_manager.py

```diff
@@ -88,13 +90,20 @@ async def send_new_transaction(
         keep_destination = whitelisted_homeserver(destination)
 
         for edu in edus:
-            context = edu.get_context()
-            if context:
-                span_contexts.append(extract_text_map(json_decoder.decode(context)))
+            tracing_context_json = edu.get_tracing_context_json()
+            if tracing_context_json:
+                context = extract_text_map(json_decoder.decode(tracing_context_json))
+                if context:
+                    span_context = get_span_context_from_context(context)
+                    if span_context:
+                        span_contexts.append(span_context)
             if keep_destination:
-                edu.strip_context()
+                edu.strip_tracing_context()
 
-        with start_active_span_follows_from("send_transaction", span_contexts):
+        with start_active_span(
+            "send_transaction",
+            links=[Link(span_context) for span_context in span_contexts],
```

Comment:
OpenTelemetry `Link`'s are the equivalent to the OpenTracing `follows_from` references. 

To give more context on why:
 - Our [previous `start_active_span_follows_from(...)`](https://github.com/matrix-org/synapse/blob/bf3115584c876392e7849cd4ba2271165572588d/synapse/logging/opentracing.py#L521) passed in the [`span_contexts` to `tracer.start_active_span(references=xxx)` as `follows_from` references](https://github.com/matrix-org/synapse/blob/bf3115584c876392e7849cd4ba2271165572588d/synapse/logging/opentracing.py#L550-L554) (`opentracing`).
 - You can see in the OpenTelemetry shim for OpenTracing that it [handles `opentracing` references as `Links` in `opentelemetry`](https://github.com/open-telemetry/opentelemetry-python/blob/c9222bfc18ec91f041c2f0a9eac8560f61dcb338/shim/opentelemetry-opentracing-shim/src/opentelemetry/shim/opentracing_shim/__init__.py#L656-L659)